kroxylicious-migrations/src/main/resources/META-INF/rewrite/rewrite.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+type: specs.openrewrite.org/v1beta/recipe
```

Comment:
shouldn't the rewrite.yaml be part of the module that they relate to?  In this case, the the kroxylicious-record-encryption